packages/app/universal-ts-utils/src/node.ts

```diff
@@ -23,6 +23,9 @@ export * from './public/object/isEmpty.js'
 export * from './public/object/pick.js'
 export * from './public/object/transformToKebabCase.js'
 
+// string
```

Comment:
can you also add readme entry for it?

Reply:
Discussed on Slack, we are not documenting methods right now but we should do it so I will address it in a separate task -> https://lokalise.atlassian.net/browse/DXP-335